param_server:
  ros__parameters:
    name:
      # publish
      grid: "/rmcs_map/map/grid"
      cost: "/rmcs_map/map/cost"
      status: "/rmcs_map/status"
      transformed_map: "/rmcs_map/transformed_map"
      # subscribe
      control:
        velocity: "/rmcs_map/control/velocity"
        rotation: "/rmcs_map/control/rotation"
        gimbal: "/rmcs_map/control/gimbal"
      lidar: "/livox/lidar"
      # other
      node: "rmcs_map"
      frame:
        map: "map_link"
        lidar: "lidar_link"
    # process config
    switch:
      # livox | pointcloud2
      pointcloud_type: "livox"
      cost_map: false
      grid_map: true
      publish_transformed_cloud: true
      publish_test: false
    lidar:
      livox_frames: 2
    grid:
      resolution: 0.05
      grid_width: 10.0
      lidar_blind: 0.8
      height_wight: 120.0
      ground_height: 0.2
    filter:
      threshold: 50
